python/gink/impl/graph.py

```diff
@@ -17,27 +17,39 @@
 class Vertex(Container):
     BEHAVIOR = VERTEX
 
-    def __init__(self, *,
-                 arche: bool = False,
-                 muid: Optional[Muid] = None,
-                 bundler: Optional[Bundler] = None,
-                 database: Optional[Database] = None):
+    def __init__(
+            self,
+            muid: Optional[Union[Muid, str]] = None,
+            *,
+            arche: Optional[bool] = None,
+            database: Optional[Database] = None,
+            bundler: Optional[Bundler] = None,
+            comment: Optional[str] = None,
+    ):
         """
         Creates a placeholder node to contain the idea of something.
 
-        muid: the global id of this vertex, created on the fly if None
-        db: database send bundles through, or last db instance created if None
+        muid: the global id of this container, created on the fly if None
+        arche: whether this will be the global version of this container (accessible by all databases)
```

Comment:
Rather than calling it the "global" version, I think I prefer the term "archetypal" version.  And saying that it's accessible by all databases is accurate though a little confusing.  Maybe "addressable by anyone without having an address?"  Not sure (and you don't necessarily need to change this PR for this, just thinking about how we'll communicate things in the future).